rhea/build/boards/altera/_de1_SoC.py

```diff
@@ -0,0 +1,229 @@
+#
```

Comment:
There seems to be two very similar definitions for the `DE1SOC`?  I assume the following is the intended one?